src/Settings.ts

```diff
@@ -59,13 +59,14 @@ export default class EditInNeovimSettingsTab extends PluginSettingTab {
           .onChange(async (value) => {
             this.plugin.settings.listenOn = value;
             await this.plugin.saveSettings();
+            new Notice("Restart Obsidian to apply the new server location");
```

Comment:
I would not include this notice, it will be obscured by the dim region behind the settings modal and difficult to read